cmd/osm-controller/osm-controller.go

```diff
@@ -239,6 +242,19 @@ func main() {
 		"/health/ready": health.ReadinessHandler(funcProbes, getHTTPHealthProbes()),
 		"/health/alive": health.LivenessHandler(funcProbes, getHTTPHealthProbes()),
 	})
+
+	webhookHandlerCert, err := certManager.IssueCertificate(
+		certificate.CommonName(fmt.Sprintf("%s.%s.svc", validatingWebhookServiceCN, osmNamespace)),
+		constants.XDSCertificateValidityPeriod)
+	if err != nil {
+		events.GenericEventRecorder().FatalEvent(err, events.CertificateIssuanceFailure, "Error issuing certificate for the validating webhook")
+	}
+
+	if cfg.GetFeatureFlags().EnableValidatingWebhook {
```

Comment:
I don't see the this flag in the meshConfig crd : https://github.com/openservicemesh/osm/blob/ceee5d3c825ef4260c4a4538899c55f4ec0e67ba/charts/osm/crds/meshconfig.yaml#L190

Reply:
You also need to add the flag here : https://github.com/openservicemesh/osm/blob/ceee5d3c825ef4260c4a4538899c55f4ec0e67ba/charts/osm/templates/preset-mesh-config.yaml#L31

Reply:
done!